pkg/actuators/awsplacementgroup/controller_test.go

```diff
@@ -0,0 +1,46 @@
+package awsplacementgroup
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+var _ = Describe("AWSPlacementGroupReconciler", func() {
+	namespace := &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{GenerateName: "mhc-test-"}}
```

Comment:
You'll need to init this inside the before each and create it on each iteration, with this, you'll overwrite the value here on the first test and then continue using the same namespace throughout

Also `mhc-test-` is probably not what you want

Reply:
Ah good catch!